file_formats/ie_formats/itm_v2.0.htm

```diff
@@ -555,18 +555,18 @@
             <td>1 (char)</td>
             <td>Timing mode
               <ul>
-                <li>0 = Duration</li>
-                <li>1 = Permanent</li>
-                <li>2 = While equipped</li>
-                <li>3 = Delayed Duration</li>
-                <li>4 = Delayed</li>
-                <li>5 = Delayed (transforms to 8)</li>
-                <li>6 = Duration?</li>
-                <li>7 = Permanent?</li>
-                <li>8 = Permanent (unsaved)</li>
-                <li>9 = Permanent (after Death)</li>
-                <li>10 = Trigger</li>
-                <li>4096 = Absolute duration</li>
+                <li>0=Duration</li>
+                <li>1=Permanent</li>
+                <li>2=While equipped</li>
+                <li>3=Delayed Duration</li>
+                <li>4=Delayed</li>
+                <li>5=Delayed (transforms to 8)</li>
+                <li>6=Duration?</li>
+                <li>7=Permanent?</li>
+                <li>8=Permanent (unsaved)</li>
+                <li>9=Permanent (after Death)</li>
+                <li>10=Trigger</li>
+                <li>4096=Absolute duration</li>
```

Comment:
This is also just less readable.

Reply:
Between the three ITM versions, I count 206 instances of the equal sign.  Of those, 125 are surrounded by spaces (" = ") which means 81 are not.  I have no particular preference (other than a desire for consistency), so I guess I'll go "with spaces".